packages/react-component-library/src/components/DatePicker/DatePicker.tsx

```diff
@@ -200,7 +200,7 @@ export const DatePicker: React.FC<DatePickerProps> = ({
                 </NavButton>
 
                 <div className={gridClasses}>
-                  {activeMonths.map((month, index) => (
+                  {activeMonths.map(month => (
```

Comment:
I suppose an option could have been:

```
{activeMonths.map({ year, month } => (
  <Month
    key={`${year}-${month}`}
    year={year}
    month={month}
    firstDayOfWeek={firstDayOfWeek}
  />
))}
```

Reply:
Yeah cool - i'll destructure I was deleting the rogue param the IDE flagged up